### ISSUE TITLE
`Command::new("clear")` panics on windows, consider using crossterm?

### ISSUE THREAD

**Issue by cndofx** [TiagoRCorreia/rtodo#12]
Windows uses `cls`, so one hacky solution would be to use conditional compilation to run `Command::new("cls")` when building for windows. Using the [crossterm](https://github.com/crossterm-rs/crossterm) crate for that instead is probably ideal, though. crossterm would also get you other features like the alternate screen and better input handling for your menus.

**Comment by TiagoRCorreia** [comment]
Yes, maybe it's a good solution. Do you want to make a pull request? I will look at the crossterm crate for more info about it. I don't have Windows on my computer for testing.

**Comment by cndofx** [comment]
Sure, I could work on that. It might be a few days at least before I find the time, though

**Comment by TiagoRCorreia** [comment]
Ok, awesome. If you need something let me know.
